docs/sphinx/source/api.rst

```diff
@@ -236,7 +236,7 @@ PV temperature models
    temperature.pvsyst_cell
    temperature.faiman
    temperature.fuentes
-   pvsystem.PVSystem.sapm_celltemp
```

Comment:
https://github.com/pvlib/pvlib-python/pull/1054#discussion_r484982981 and #1055 

I'd leave it here for now.